src/lib/utils/hooks/useCurationLists.ts

```diff
@@ -1,26 +1,34 @@
 import { useCustomStoreQueryParameter } from "../../../pages/custom-store/useCustomStoreQueryParameter";
 import { CONFIG } from "../../config";
 import { parseCurationList } from "../curationList";
+import { useSellerBlacklist } from "./useSellerBlacklist";
 
 export function useCurationLists() {
-  const withOwnProducts = useCustomStoreQueryParameter("withOwnProducts");
-  const sellerCurationListFromUrl =
+  const sellerBlacklist = useSellerBlacklist({
+    sellerBlacklistUrl: CONFIG.sellerBlacklistUrl,
+    allowConnectedSeller: false
+  });
+  const sellerCurationList = sellerBlacklist.isSuccess
+    ? (sellerBlacklist.curatedSellerIds as string[])
+    : [];
+  const sellerCurationListFromUrlParam =
     useCustomStoreQueryParameter("sellerCurationList");
+  const sellerCurationListFromUrl = sellerCurationListFromUrlParam
+    ? parseCurationList(sellerCurationListFromUrlParam)
+    : [];
   const offerCurationListFromUrl =
     useCustomStoreQueryParameter("offerCurationList");
 
   return {
-    enableCurationLists:
-      withOwnProducts === "all"
-        ? false
-        : ["mine", "custom"].includes(withOwnProducts)
-        ? true
-        : CONFIG.enableCurationLists,
-    sellerCurationList: sellerCurationListFromUrl
-      ? parseCurationList(sellerCurationListFromUrl)
-      : CONFIG.sellerCurationList,
+    enableCurationLists: CONFIG.enableCurationLists, // curation list is always enabled except if env var is used to disable it
```

Comment:
I think this comment would make sense if it was hardcoded to true? so please remove the comment if my understanding is correct